native/cocos/2d/renderer/RenderDrawInfo.h

```diff
@@ -0,0 +1,130 @@
+#pragma once
+#include <cocos/2d/renderer/UIMeshBuffer.h>
+#include <cocos/base/TypeDef.h>
+#include <cocos/core/assets/Material.h>
+#include <cocos/core/scene-graph/Node.h>
+#include <cocos/renderer/gfx-base/states/GFXSampler.h>
+#include <math/Color.h>
+#include <math/Vec2.h>
+#include <math/Vec3.h>
+#include <math/Vec4.h>
+#include <vector>
+
+namespace cc {
+struct Render2dLayout {
+    cc::Vec3 position;
+    cc::Vec2 uv;
+    cc::Vec4 color;
```

Comment:
No need to add `cc::` prefix since it's already inside cc namespace .

Reply:
done